src/cpp/rtps/common/GuidUtils.hpp

```diff
@@ -102,7 +102,8 @@ class GuidUtils
         prefix_.value[4] = static_cast<octet>(pid & 0xFF);
         prefix_.value[5] = static_cast<octet>((pid >> 8) & 0xFF);
 
-        std::random_device generator;
+        std::random_device device;
```

Comment:
Maybe a comment may help in the future, the reason of using this random generator or maybe the number of the issue.

Reply:
Done